contrib/win32/openssh/AppveyorHelper.psm1

```diff
@@ -266,17 +282,17 @@ function Publish-Artifact
     [System.Collections.ArrayList] $artifacts = new-object System.Collections.ArrayList
     
     # Get the build.log file for each build configuration        
-    Add-BuildLog -artifacts $artifacts -buildLog (Get-BuildLogFile -root $repoRoot.FullName -Configuration Release -NativeHostArch x64)
-    Add-BuildLog -artifacts $artifacts -buildLog (Get-BuildLogFile -root $repoRoot.FullName -Configuration Release -NativeHostArch x86)
+    Add-BuildLog -artifacts $artifacts -buildLog (Get-BuildLogFile -root $repoRoot.FullName -Configuration Release)
+    #Add-BuildLog -artifacts $artifacts -buildLog (Get-BuildLogFile -root $repoRoot.FullName -Configuration Release -NativeHostArch x86)
 
-    if($Global:OpenSSHTestInfo)
+    <#if($Global:OpenSSHTestInfo)
```

Comment:
We could have all tests write output to a specific folder and later upload all contents of that folder as part of CI run.